python_modules/dagster/dagster_tests/execution_tests/test_metadata.py

```diff
@@ -382,3 +383,14 @@ def foo_asset():
     assert build_node_defs_snapshot(
         Definitions(assets=[foo_asset]).get_implicit_global_asset_job_def()
     )
+
+
+def test_thing():
```

Comment:
If we're going to keep this, can we rename it and maybe put a real assertion that there is no warning?